astropy/io/votable/tests/test_table.py

```diff
@@ -261,6 +261,56 @@ def test_write_with_format():
     assert b"TABLEDATA" not in obuff
 
 
+def test_read_write_votable_parquet(tmp_path):
```

Comment:
pyarrow is needed for this test to run, so a conditional should test for its presence.